testing/prepopulate.nohup

```diff
@@ -16,6 +16,8 @@ generate_ssh_key() {
     ssh-keygen -t rsa -f $ssh_key -q -N ""
   fi
 
+  rm -rf $HOME/.ssh/known_hosts
```

Comment:
When is this executed again?

Reply:
Somehow the home directory is still persisted it seems.  I didn't figure out why.  Without this command you'll get a host key verification error on the next start of the container.